Cue/app/CueNavigator.js

```diff
@@ -107,6 +108,8 @@ var CueNavigator = React.createClass({
       return <PlayDeckView navigator={navigator} deck={route.playDeck} {...route} />
     } else if (route.preview) {
       return <DeckPreview navigator={navigator} deck={route.preview}/>
+    } else if (route.failedSyncs) {
+      return <SyncConflict navigator={navigator} failedSyncs={route.failedSyncs} />
```

Comment:
Minor nitpick, but since this is supposed to be a modal screen, it should be configured on iOS to use `Navigator.SceneConfigs.FloatFromBottom` with swipe-to-dismiss gesture disabled like `PlayDeckSetupView` in `configureScene`.